src/imex.rs

```diff
@@ -472,7 +473,54 @@ async fn import_backup(context: &Context, backup_to_import: impl AsRef<Path>) ->
             .query_row(
                 "SELECT file_name, file_content FROM backup_blobs WHERE id = ?",
                 paramsv![file_id],
-                |row| Ok((row.get::<_, String>(0)?, row.get::<_, Vec<u8>>(1)?)),
+                // cs: This is the affected line NOW for producing the error. But
```

Comment:
Just remove the old code, with the correct code it is quite clear why previously used code is incorrect. Maybe leave a comment around `ValueRef::Blob` instead saying that it is for invalid UTF-8 filenames.

Reply:
Ok ... can do that. Normally I like to leave comments for later understanding of code.